sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/azure/eventhub/extensions/checkpointstoreblobaio/_vendor/storage/blob/aio/_blob_client_async.py

```diff
@@ -927,7 +927,7 @@ async def start_copy_from_url(self, source_url, metadata=None, incremental_copy=
                 :caption: Copy a blob from a URL.
         """
         options = self._start_copy_from_url_options(
-            source_url,
+            source_url=self._encode_source_url(source_url),
```

Comment:
This file is not in the storage library. I guess you wanted to change the file in [the storage SDK](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_client_async.py)?

Reply:
Oh yea! thank you for pointing it out